account_credit_control/models/credit_control_communication.py

```diff
@@ -194,26 +194,30 @@ def _get_credit_control_communication_table(self):
         table_content = "<br/><h3>%s</h3>" % _("Invoices summary")
         table_content += "<table style='%s'><tr>%s</tr>" % (table_style, tr_content)
         for line in self.credit_control_line_ids:
-            tr_content = "<td style='%s'>%s</td>" % (th_style, line.invoice_id.name)
+            if line.invoice_id.move_type == "entry":
+                name = line.move_line_id.ref
+            else:
+                name = line.invoice_id.name
+            tr_content = "<td style='%s'>%s</td>" % (th_style, name)
             tr_content += "<td style='%s'>%s</td>" % (
                 th_style,
                 line.invoice_id.payment_reference or "",
             )
             tr_content += "<td style='%s'>%s</td>" % (
                 th_style,
-                format_date(self.env, line.invoice_id.invoice_date),
+                format_date(self.env, line.date_entry),
             )
             tr_content += "<td style='%s'>%s</td>" % (
                 th_style,
                 format_date(self.env, line.date_due),
             )
             tr_content += "<td style='%s'>%s</td>" % (
                 th_style,
-                format_amount(self.env, line.invoice_id.amount_total, line.currency_id),
+                format_amount(self.env, line.amount_due, line.currency_id),
```

Comment:
This is not correct. It should be the total of the line.

Reply:
This change is precisely to indicate the same data shown in the report. https://github.com/OCA/credit-control/blob/df877b848a5653b2b83acb7bb80b4c2e78f15f67/account_credit_control/report/report_credit_control_summary.xml#L58

Reply:
Uhm, it sounds weird. Both are the residual. Where is the total expressed?

Reply:
Only the `amount_due` and `balance_due` fields exist, `amount_due` seems to represent the appropriate value. https://github.com/OCA/credit-control/blob/df877b848a5653b2b83acb7bb80b4c2e78f15f67/account_credit_control/models/credit_control_line.py#L181